e2e/playwright.config.ts

```diff
@@ -0,0 +1,50 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
Why are this and the block on line 45 commented out? Are they used for running the e2e tests locally? If yes, maybe the comments should mention it?

Reply:
That's what came from scaffolding playwright & I thought I'll leave it in for the next iteration we do, so the information is available right in our repo instead of having to look it up again.